### PR TITLE
Adapt to Dekorate

### PR DESCRIPTION
**Pull request by jacobdotcosta**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -75,15 +79,110 @@
...
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-web</artifactId>
+      <exclusions>
```
Comment (Sgitario) [inline pom.xml]: Why do we need so many exclusions? It's quite unexpected for me to having to add them just to generate the Kubernetes resources using Dekorate.

**Review by Sgitario — CHANGES_REQUESTED** [review]

**Review by Sgitario — CHANGES_REQUESTED** [review]
Same comments as in https://github.com/snowdrop/health-check-example/pull/82#pullrequestreview-1100595213

File: pom.xml (hunk 1 above)
Comment (jacobdotcosta) [reply pom.xml]: fixed

**Review by Sgitario — APPROVED** [review]
lgtm now

**Pull request merged**